scripts/renderer.py

```diff
@@ -42,7 +42,7 @@ def render_local_viewing(article):
 
     # Give local references to all images and article references.
     for line in original.readlines():
-        fix_lines(line, fixed_file, "local_viewing")
+        fix_lines(line, fixed_file, "local_viewing", cloudfront_url)
```

Comment:
Should be `fix_lines(line, fixed_file, "local_viewing", None)

Alternatively set the default to the last param of fix_lines to None.

Admittedly, the interface provided by fix_lines isn't clear. Read the function -- if you set the third param to "local_viewing", the 4th param isn't required.